cmd/kubeadm/app/constants/constants.go

```diff
@@ -249,6 +249,10 @@ const (
 	AuditPolicyDir = "audit"
 	// AuditPolicyFile is the name of the audit policy file itself
 	AuditPolicyFile = "audit.yaml"
+	// KubeAuditWebhookConfigVolumeName is the name of the volume that will contain the audit webhook config
+	KubeAuditWebhookConfigVolumeName = "audit-webhook-config"
```

Comment:
Do we actually _need_ this subdir or could it be statically mounted right into `/etc/kubernetes`?

Reply:
We're reusing the existing audit policy subdirectory.
see cmd/kubeadm/app/constants/constants.go:380
```go
func GetStaticPodAuditWebhookConfigFile() string {
     return filepath.Join(KubernetesDir, **AuditPolicyDir**, AuditWebhookConfigFile)
}
```